src/compas_fab/robots/constraints.py

```diff
@@ -154,35 +154,38 @@ class JointConstraint(Constraint):
         The name of the joint this contraint refers to.
     value: float
         The targeted value for that joint.
-    tolerance: float
-        The bound to be achieved is [value - tolerance, value + tolerance]
+    tolerance_above: float
+    tolerance_below: float
```

Comment:
could you add here a description as well?

Reply:
added in ab82257